ckanext/unaids/theme/templates/base.html

```diff
@@ -1,11 +1,7 @@
 {% ckan_extends %}
 
-{% block styles %}
-  {{ super() }}
-  {% asset 'ckanext-unaids/UnaidsCSS' %}
-{% endblock %}
-
 {% block head_extras %}
+  {% asset 'ckanext-unaids/UnaidsCSS' %}
   {{ super() }}
```

Comment:
The reason I had to do this was that `recline_view.html` overwrites the `{% block styles %}` block in the template
https://github.com/ckan/ckan/blob/master/ckanext/reclineview/theme/templates/recline_view.html

So we need to make sure our css fixes are still applied.

An alternative approach would be to make a specific `recline_view_fixes.css` and apply it to `{% block head_extras %}` but I think applying the whole of `/unaids/assets/custom.css` is still ok